auto_round/calib_dataset.py

```diff
@@ -102,9 +102,50 @@ def get_pile_dataset(tokenizer, seqlen, dataset_name="NeelNanda/pile-10k", split
     from datasets import load_dataset
 
     split = "train"
+    
     tokenizer_function = get_tokenizer_function(tokenizer, seqlen, apply_chat_template=apply_chat_template)
+    try:
+        calib_dataset = load_dataset(dataset_name, split=split)
+    except Exception as e:
+        logger.error(f"Failed to load the dataset: {e}." \
+                       "Another source dataset can be used as an alternative by installing the 'modelscope' library and " \
```

Comment:
Consider using  a backup dataset by "pip install modelscope" and set xxx in AutoRound API

Reply:
better have a check of the accuracy for some models

Reply:
If the error message is too lengthy, display our message on a new line